### PR TITLE
trivial: -llmq-qvvec-sync help description

### PR DESCRIPTION
**Pull request by thephez**
Fix parentheses

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -617,7 +617,7 @@ void SetupServerArgs()
...
-    gArgs.AddArg("-llmq-qvvec-sync=<quorum_name:mode>", strprintf("Defines from which LLMQ type the masternode should sync quorum verification vectors. Can be used multiple times with different LLMQ types. (mode: %d (sync always from all quorums of the type defined by \"quorum_name\"), %d (sync from all quorums of the type defined by \"quorum_name\") if member of any of the quorums", (int32_t)llmq::QvvecSyncMode::Always, (int32_t)llmq::QvvecSyncMode::OnlyIfTypeMember), false, OptionsCategory::MASTERNODE);
+    gArgs.AddArg("-llmq-qvvec-sync=<quorum_name:mode>", strprintf("Defines from which LLMQ type the masternode should sync quorum verification vectors. Can be used multiple times with different LLMQ types. Mode: %d (sync always from all quorums of the type defined by \"quorum_name\"), %d (sync from all quorums of the type defined by \"quorum_name\" if a member of any of the quorums)", (int32_t)llmq::QvvecSyncMode::Always, (int32_t)llmq::QvvecSyncMode::OnlyIfTypeMember), false, OptionsCategory::MASTERNODE);
```
Comment (xdustinface) [inline src/init.cpp:620]: nit: Hmm, the `mode` there was a reference the one in here `-llmq-qvvec-sync=<quorum_name:mode>`. Im not sure how i feel about having it to start with `M` but also starting with `m` after `.` seems weird.. 🙈 Thoughts?

Comment (UdjinM6) [reply src/init.cpp:620]:
```suggestion
    gArgs.AddArg("-llmq-qvvec-sync=<quorum_name>:<mode>", strprintf("Defines from which LLMQ type the masternode should sync quorum verification vectors. Can be used multiple times with different LLMQ types. <mode>: %d (sync always from all quorums of the type defined by <quorum_name>), %d (sync from all quorums of the type defined by <quorum_name> if a member of any of the quorums)", (int32_t)llmq::QvvecSyncMode::Always, (int32_t)llmq::QvvecSyncMode::OnlyIfTypeMember), false, OptionsCategory::MASTERNODE);
```

**Review by xdustinface — APPROVED** [review]
utACK

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]

**Pull request merged**